server/mailer/index.js

```diff
@@ -0,0 +1,35 @@
+const SendGridMail = require('@sendgrid/mail');
+
+const mailTemplate = require('./mail-template');
+
+const { SENDGRID_API_KEY } = process.env;
+
+SendGridMail.setApiKey(SENDGRID_API_KEY);
+
+const sendInvitation = (req, resp) => {
+  const { email: receiver } = req.body;
```

Comment:
only wonder: shouldn't we sanitize an email address here?

Reply:
I think we don't need to sanitize data as long as we're not making any query to db. 
But I think we should create task for email validation.